packages/keyboard-keys/src/keyCodes.ts

```diff
@@ -0,0 +1,163 @@
+export const Cancel = 3;
```

Comment:
how about using `as const` here for typings?

Reply:
yeah sounds good, need another round of my generation script :D

Reply:
no need `const` on primitives are not widened 

```
 const Cancel = 3; 

// is same as

 const Cancel = 3 as const;

```

Reply:
> need another round of my generation script :D

would you mind to make this tool a part of our repo ?

Reply:
@Hotell thx for the ts expertise :)

The generation script is just some web scraping I did from the w3c spec site, nothing to be added to the repo

Reply:
> The generation script is just some web scraping

to me that sounds very appealing to have proper automation for our consumers ;) I'll be happy to help get that in